packages/components/src/Card/tests/Card.test.tsx

```diff
@@ -109,3 +109,14 @@ test("it should should be clickable if it's clickable", () => {
 
   expect(clickHandler).toHaveBeenCalledTimes(1);
 });
+
+it("renders an external link card", () => {
+  const tree = renderer
+    .create(
+      <Card url="https://frend.space" external={true}>
+        <p>This is a link card.</p>
+      </Card>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
Could you test by getting the attribute specifically? You could use React testing library to get the anchor tag, and then getting the target attribute and confirming it's "_target"? 

Snapshots can work, but might be best to test specifically for the attribute.

Reply:
Done, let me know if it looks good